BEIMA.Backend/StorageService/AzureStorageProvider.cs

```diff
@@ -0,0 +1,116 @@
+﻿using Azure.Storage.Blobs;
+using Azure.Storage.Sas;
+using Microsoft.AspNetCore.Http;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace BEIMA.Backend.StorageService
+{
+    public sealed class AzureStorageProvider : IStorageProvider
+    {
+        private static BlobContainerClient containerClient;
+
+        public AzureStorageProvider()
```

Comment:
It may seem redundant because of the interface, but we should still use method and class headers here. Most of them you should just be able to copy from the interface.